gadgets/advise_networkpolicy/test/integration/advise_networkpolicy_test.go

```diff
@@ -32,7 +32,6 @@ import (
 var expectedYaml = `apiVersion: networking.k8s.io/v1
 kind: NetworkPolicy
 metadata:
-  creationTimestamp: null
```

Comment:
I see that upstream also dropped creationTimestamp=null in tests when switching to k8s.io/api v0.34.0:
https://github.com/kubernetes/apimachinery/commit/9386fa7b711240e7f634bcca2e2994961ca8bbe6

This comes from:
https://github.com/kubernetes/kubernetes/pull/130989